main.c

```diff
@@ -51,14 +52,21 @@ main(int argc, char *argv[])
 	int		force_update = 0;
 	char		**pkgargs = NULL;
 	const char	*chrootpath = NULL;
+	int		v4flag = 0, v6flag = 0, ffi = 0;
 
 	setprogname("pkgin");
 
 	/* Default to not doing \r printouts if we don't send to a tty */
 	parsable = !isatty(fileno(stdout));
 
-	while ((ch = getopt(argc, argv, "dhyfFPvVl:nc:t:p")) != -1) {
+	while ((ch = getopt(argc, argv, "46dhyfFPvVl:nc:t:p")) != -1) {
 		switch (ch) {
+		case '4':
+			v4flag = 1;
+			break;
+		case '6':
+			v4flag = 1;
```

Comment:
This was probably meant to be:
```suggestion
			v6flag = 1;
```